aggregate/aggregate_it_test.go

```diff
@@ -85,6 +86,7 @@ func TestDistinctValuesAll(t *testing.T) {
 		config.Serialization.SetPortableFactories(it.SamplePortableFactory{})
 	}
 	it.MapTesterWithConfig(t, cbCallback, func(t *testing.T, m *hz.Map) {
+		it.SkipIf(t, "hz <= 4.0.3")
```

Comment:
Maybe we should change the condition to: `hz < 4.1`.

Reply:
Addressed in e8aff4dd045b95ef5195f09d84a7a73a00af6e1b